content/en/agent/faq/agent-downgrade-major.md

```diff
@@ -1,17 +1,70 @@
+| CentOS       | `DD_AGENT_MAJOR_VERSION=6 DD_API_KEY="<DATADOG_API_KEY>" bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"` |
+| Debian       | `DD_AGENT_MAJOR_VERSION=6 DD_API_KEY="<DATADOG_API_KEY>" bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"` |
+| Fedora       | `DD_AGENT_MAJOR_VERSION=6 DD_API_KEY="<DATADOG_API_KEY>" bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"` |
+| Red Hat      | `DD_AGENT_MAJOR_VERSION=6 DD_API_KEY="<DATADOG_API_KEY>" bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"` |
+| Ubuntu       | `DD_AGENT_MAJOR_VERSION=6 DD_API_KEY="<DATADOG_API_KEY>" bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"` |
+| SUSE         | `DD_AGENT_MAJOR_VERSION=6 DD_API_KEY="<DATADOG_API_KEY>" bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"` |
+
+[1]: /agent/faq/how-do-i-uninstall-the-agent
+[2]: /agent/versions/upgrade_to_agent_v6
+{{% /tab %}}
+{{% tab "Windows" %}}
+
+1. [Uninstall Agent v7 from your system][1].
+2. [Download the Datadog Agent installer][2].
+3. Run the installer (as **Administrator**) by opening `datadog-agent-6-latest.amd64.msi`.
+4. Follow the prompts, accept the license agreement, and enter your [Datadog API key][3].
+5. When the install finishes, you are given the option to launch the Datadog Agent Manager.
+
+**Note**: Links to all available versions of the Windows Installer are [provided in JSON format][4].
+
+[1]: /agent/faq/how-do-i-uninstall-the-agent
+[2]: https://ddagent-windows-stable.s3.amazonaws.com/datadog-agent-6-latest.amd64.msi
+[3]: https://app.datadoghq.com/account/settings#api
+[4]: https://s3.amazonaws.com/ddagent-windows-stable/installers.json
+{{% /tab %}}
+{{% tab "MacOS" %}}
+
+First, [uninstall Agent v7 from your system][1].
+
+Then, if you followed the instructions to [upgrade from v6 to v7][2], run the Agent installation command with the environment variable `DD_AGENT_MAJOR_VERSION=6` in order to downgrade your Agent from version 7 to version 6:
+
+```shell
+DD_AGENT_MAJOR_VERSION=6 DD_API_KEY="<DATADOG_API_KEY>" bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_mac_os.sh)"
+```
+
+[1]: /agent/faq/how-do-i-uninstall-the-agent
+[2]: /agent/versions/upgrade_to_agent_v6
+{{% /tab %}}
+{{< /tabs >}}
+
+## Downgrade the Agent from v6 to v5
+
+This guide assumes you upgraded to the Agent v6 using our [upgrade guide][1], choose your OS to see the detailed instructions on how to downgrade your Agent from version 6 to version 5:
```

Comment:
Note: the v6 -> v5 downgrade instructions currently do not work, as they expect the v6 repo to be in a `datadog-beta.list` (or `datadog-beta.repo`) file, which was only the case when v6 was still in beta.
This should be fixed in another PR.